package.json

```diff
@@ -171,6 +171,7 @@
     "test:acceptance:server": "concurrently --kill-others \"npm run verdaccio:server\" \"npm run test:acceptance\"",
     "test:e2e": "cross-env BABEL_ENV=test jest --config ./test/jest.config.e2e.js",
     "test": "cross-env NODE_ENV=test BABEL_ENV=test TZ=UTC jest --config ./jest/jest.config.js --maxWorkers 2 --passWithNoTests",
+    "test:snapshopts": "npm run test -- -u",
```

Comment:
The script name tells me more to run snapshot tests than update. Also, snapshoPts is misspelled.

Reply:
make sense, updated to `test:update-snapshot`

Reply:
I would rather go test:update and using yarn instead npm

Reply:
But we use `npm ...`in other scripts too. If this should be changed to `yarn`, shouldn't it be for all or for more?

Reply:
We should migrate all scripts to use one package manager, but that is other issue, let's open a new one and keep this simple :wink:

Reply:
Exacctly.